docs/modules/ROOT/pages/index.adoc

```diff
@@ -2,11 +2,9 @@
 :!page-pagination:
 :description: Hazelcast Platform uniquely combines a distributed compute engine and a fast data store in one runtime. It offers unmatched performance, resilience and scale for real-time and AI-driven applications. 
 
-{description} 
-
 Hazelcast is a distributed computation and storage platform for consistently
 low-latency querying, aggregation and stateful computation against event
-streams and traditional data sources. It allows you to quickly build
+streams and traditional data sources. {description} It allows you to quickly build
```

Comment:
interesting, I've never seen {description} used twice on a page; does this have any implications?

Reply:
I took the first instance out and just plopped it in the middle of the existing description. Made more sense rather than looking like two descriptions but will get revised no doubt later in this cycle when we have a landing page template.